barretenberg/cpp/src/barretenberg/relations/memory_relation.hpp

```diff
@@ -131,10 +131,14 @@ template <typename FF_> class MemoryRelationImpl {
          *
          **/
 
+        /**
+         * NOTE: all of the degree calculations here are the straightforward ones. Formerly, we had modified degree
```

Comment:
I generally try to avoid making references to how things used to be in code unless its useful for understanding something that would otherwise be unexpected